### PR TITLE
Views tests and some fixes

### PR DESCRIPTION
**Pull request by yusufsafak**
#983 again rebased and fixed conflicts :) 
 ! [rejected]        views-tests -> views-tests (non-fast-forward)
I could not manage to fix this error when rebasing and pushing it :( any recommendation to fix it?

### CODE REVIEW

File: packages/node_modules/cerebral/src/views/View.js [hunk 1]
```diff
@@ -324,16 +324,7 @@ class View {
           } else {
             currentProps[key] = value
           }
-        } else if (tag.type === 'signal') {
```
Comment (yusufsafak) [inline packages/node_modules/cerebral/src/views/View.js]: no need to throw anymore!

**Comment by christianalfoni** [comment]
@yusufsafak How are we with this now? You want me to review it? :)

**Comment by yusufsafak** [comment]
@christianalfoni tests are good but lots of conflicts and changes please review it :)

File: packages/node_modules/cerebral/src/views/View.js [hunk 2]
```diff
@@ -19,14 +18,26 @@ class View {
...
     this.stateGetter = this.stateGetter.bind(this)
     this.signalGetter = this.signalGetter.bind(this)
     this.mergeProps = mergeProps
     this.controller = controller
     this._displayName = displayName
     this._hasWarnedBigComponent = false
     this.isUnmounted = false
-    this.updateComponent = onUpdate
+    this.updateComponent = onUpdate || noop
+    this._verifyPropsWarned = false
```
Comment (yusufsafak) [inline packages/node_modules/cerebral/src/views/View.js]: is it OK? should warn only one time for each component ?

**Comment by Guria** [comment]
cc @christianalfoni

File: packages/node_modules/cerebral/src/views/View.js (hunk 2 above)
Comment (christianalfoni) [inline packages/node_modules/cerebral/src/views/View.js]: Jup, only one time :)

**Pull request merged**